src/app/App.jsx

```diff
@@ -68,20 +61,26 @@ const propTypes = {
     pathname: PropTypes.string,
   }),
 };
-const appConfig = siteConfig.appConfig;
+
 const defaultProps = {
-  appTitle: appConfig.title,
-  appLogoSrc: appConfig.logoSrc,
-  hideBidiUtility: !appConfig.bidirectional,
-  defaultDir: appConfig.defaultDirection,
-  defaultTheme: appConfig.defaultTheme,
-  themes: appConfig.themes,
-  defaultLocale: appConfig.defaultLocale,
-  locales: appConfig.locales,
+  nameConfig: undefined,
```

Comment:
Do we need these defaults? Undefined is the implicit default value.

Reply:
linter complains without it

Reply:
😞

Reply:
This rule is disabled in core etc, but not here.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md